modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusStateRemover.scala

```diff
@@ -0,0 +1,71 @@
+package org.tessellation.sdk.infrastructure.consensus
+
+import cats.Show
+import cats.effect.Async
+import cats.kernel.Next
+import cats.syntax.all._
```

Comment:
in the whole project we try to import only required syntax, instead of all

Reply:
I can change it in this file since it doesn't use much syntax, but I think we should be importing `cats.syntax.all._` when there are more than a few specific `cats.syntax` imports. Even cats do it internally. It simplifies development and solves potential conflicts in imports.

Reply:
@tbekas what conflicts do you mean?

Reply:
> The reason for hand-optimizing your imports is that you limit the implicit scopes in your Scala files, reducing compilation time.

this argument convinces me to make this little effort and import only needed syntax and instances, instead of importing all everything, especially in such a large codebase we have

Reply:
I'm completely fine with trading the compiler's time for my own time. 

With hand optimizations, we need to know exactly what import has to be added for the syntax we want to use - which is problematic when something is rarely used, like `leftWiden` that is used in this file. If you don't know it you need to spend additional time searching for the right import. Also if someone isn't very proficient with cats API yet, it's nice to have a menu of methods available on hand.

>@tbekas what conflicts do you mean?

In the source of your quote, there's a next paragraph that explains it very well
https://blog.softwaremill.com/9-tips-about-using-cats-in-scala-you-might-want-to-know-e1bafd365f88

```
//we don't remember where `pure` is and
//we're trying to be smart
import cats.implicits._
import cats.instances.option._

"a".pure[Option]
//could not find implicit value for parameter F: cats.Applicative[Option]
```
>This happens because cats.implicits and cats.instances.option both extend cats.instances.OptionInstances. We’re essentially importing the implicit scope for that twice, and this confuses the compiler.